### PR TITLE
Receive udp

### PR DESCRIPTION
**Pull request by jeffgrunewald**
Ready to start getting feedback on the udp socket

Still need to plumb out the interactions of the actual service and probably extract the `test/support` stuff for accept_udp into the testing library to allow pulling the source socket into the platform runner/e2e tests as well.

### CODE REVIEW

File: apps/service_receive/lib/receive/acception/store.ex [hunk 1]
```diff
@@ -0,0 +1,33 @@
+defmodule Receive.Acception.Store do
+  @instance Receive.Application.instance()
+  @collection "acceptions"
```
Comment (jdenen) [inline apps/service_receive/lib/receive/acception/store.ex]: This... hurts to read. It just, is not a word that should ever exist.

Comment (jdenen) [reply apps/service_receive/lib/receive/acception/store.ex]: `accepting`? `acceptables`?

File: .tool-versions [hunk 2]
```diff
@@ -1,3 +1,4 @@
 erlang 22.2.1
+erlang 22.2.4
```
[the remark this answers is not in the document]
Comment (jeffgrunewald) [reply .tool-versions]: nope

File: apps/service_receive/lib/receive/acception/store.ex (hunk 1 above)
Comment (jeffgrunewald) [reply apps/service_receive/lib/receive/acception/store.ex]: changed to `accepts`

**Comment by jeffgrunewald** [comment]
Well played, @jdenen

**Pull request merged**